ftw/statusmap/utils.py

```diff
@@ -5,15 +5,28 @@ def getTransitionsForItem(wf_tool, brains, dicts):
     for index, brain in enumerate(brains):
         obj = brain.getObject()
         actions = wf_tool.listActionInfos(object=obj)
+        old_review_state_title = ''
         avail_actions = []
         for action in actions:
             if action['category'] == 'workflow':
-                avail_actions.append({
-                    'id': action['id'],
-                    'title': action['title'],
-                    'old_review_state': brain.review_state,
-                    'new_review_state': action.get('transition').new_state_id,
-                    })
+                # Construct a dict where the key is the id of the state
+                # and its value is the human readable title of the state.
+                transition = action.get('transition')
+                review_state_titles = {state[0]: state[1].title for state
```

Comment:
@maethu, thank you for the hint. I have opted to rewrite the construction of the dict.